### PR TITLE
add TLDR instructions to top-level README

### PR DESCRIPTION
**Pull request by andrewmwells-amazon**
*Issue #, if available:*

*Description of changes:*
add TLDR instructions to top-level README

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,7 +3,19 @@
 This repository contains the source code for a Java package `CedarJava` that supports using the [Cedar](https://www.cedarpolicy.com) policy language.
 It also contains source code for a Rust crate `CedarJavaFFI` that enables calling Cedar library functions (written in Rust) from Java.
 
-You can find build instructions and more information in the subfolders.
+
+## Getting Started
+You can find detailed build instructions and more information in the subfolders.
```
Comment (shaobo-he-aws) [inline README.md:8]: Is it easy to add links to them here?

**Review by shaobo-he-aws — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (andrewmwells-amazon) [reply README.md:8]: Do you mean links to the local files or the files on github?

**Review by cdisselkoen — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (shaobo-he-aws) [reply README.md:8]: Yes.

Comment (andrewmwells-amazon) [inline README.md:8]:
```suggestion
You can find detailed build instructions and more information in the subfolders [CedarJavaFFI](https://github.com/cedar-policy/cedar-java/blob/main/CedarJavaFFI/README.md), [CedarJava](https://github.com/cedar-policy/cedar-java/blob/main/CedarJava/README.md).
```

**Review by shaobo-he-aws — APPROVED** [review]

**Pull request merged**